includes/class-amp-travel-blocks.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * AMP Travel blocks class.
+ *
+ * @package WPAMPTheme
+ */
+
+/**
+ * Class AMP_Travel_Blocks.
+ *
+ * @package WPAMPTheme
+ */
+class AMP_Travel_Blocks {
+
+	/**
+	 * AMP_Travel_Blocks constructor.
+	 */
+	public function __construct() {
+		if ( function_exists( 'gutenberg_init' ) ) {
+			add_action( 'enqueue_block_editor_assets', array( $this, 'enqueue_editor_scripts' ) );
+			add_filter( 'the_content', array( $this, 'filter_the_content_amp_atts' ), 10, 1 );
+			add_filter( 'wp_kses_allowed_html', array( $this, 'filter_wp_kses_allowed_html' ), 10, 2 );
+			add_filter( 'pre_get_posts', array( $this, 'filter_search_pre_get_posts' ), 10, 1 );
```

Comment:
This should [probably be](https://codex.wordpress.org/Plugin_API/Action_Reference/pre_get_posts) `add_action( 'pre_get_posts'`, instead of `add_filter`.

Also, the 3rd argument of `1` doesn't hurt, but this would probably be cleaner without it.